lib/plain/entities/extension.ts

```diff
@@ -0,0 +1,177 @@
+import { RawAxiosRequestHeaders } from 'axios'
+import {
+  CollectionProp,
+  GetExtensionParams,
+  GetSpaceEnvironmentParams,
+  QueryParams,
+} from '../../common-types'
+import { CreateExtensionProps, ExtensionProps } from '../../entities/extension'
+import { OptionalDefaults } from '../wrappers/wrap'
+
+export type ExtensionPlainClientAPI = {
+  /**
+   * Fetches the Extension
+   * @param params entity IDs to identify the Extension
+   * @returns the Extension and its metadata
+   * @throws if the request fails, or the Extension is not found
+   * @example
+   * ```javascript
+   * const uiExtension = await client.extension.get({
+   *   spaceId: '<space_id>',
+   *   environmentId: '<environment_id>',
+   *   extensionId: '<extension_id>',
+   * });
+   * ```
+   */
+  get(params: OptionalDefaults<GetExtensionParams & QueryParams>): Promise<ExtensionProps>
+  /**
+   * Fetches all Extensions of a Space
```

Comment:
is this supposed to be `Space` or `Environment`? the next line refers to `Environment`

Reply:
good call, it is using `Space/Environment` elsewhere, switched to that